modules/salt/pages/yomi.adoc

```diff
@@ -120,6 +120,7 @@ cobbler distro add \
   --kernel-options='rd.kiwi.install.pxe rd.kiwi.install.image=tftp://server-address/pxe-yomi-image/image.xz rd.kiwi.ramdisk ramdisk_size=1572864 net.ifnames=1'
 ----
 
+By default the ``salt-minion`` available in the ``pxe-yomi-image`` is configured to find the master in the ``salt`` address. If the DNS server do not resolve this address, you need to adjust the ``kernel-options`` parameter from the previous Cobbler command, to add a new kernel command line: ``master=my_master_address``. This will overwrite the default configuration for the ``salt-minion``.
```

Comment:
```suggestion
By default, the ``salt-minion`` parameter in ``pxe-yomi-image`` is configured to find the Salt master in the lcoation provided at the ``salt`` parameter. 
If the DNS server is not able to resolve this address, you need to adjust the ``kernel-options`` parameter from the previous Cobbler command, and add a new kernel command line of ``master=my_master_address``. 
This will overwrite the default configuration for the ``salt-minion``.
```

Not sure I've understood this properly, so I might have changed your meaning. Also, it's a little confusing referring to the previous step, can we either move this to that step, or perhaps add it at the end (after the procedure) as a troubleshooting note instead?

Reply:
Let me commit this change and add some fix on top

Reply:
Done